torch_xla/csrc/helpers.cpp

```diff
@@ -322,6 +329,116 @@ xla::XlaOp XlaHelpers::DynamicReshapeAs(xla::XlaOp input,
              : xla::Reshape(input, shape.dimensions());
 }
 
+#if EXPERIMENTAL_XLA_UNBOUNDED_DYNAMISM
```

Comment:
not a fan of macro, if you want to guard these functions you can add a check like `XLA_CHECK(getenv(EXPERIMENTAL_XLA_UNBOUNDED_DYNAMISM))`

Reply:
sg, will update